content/chapter-03/ApplicationMetrics.md

```diff
@@ -9,13 +9,13 @@
   * Availability percentage, e.g.: Percentage of uptime for interval of time
   * Concurrent users, e.g.: Concurrent user count distribution over a day
   * User satisfaction using [Apdex score](https://www.apdex.org/)
-  * [Throughput](https://www.cdnetworks.com/en/news/6-critical-web-application-performance-metrics-to-consider/4257). This is very tight to the application at hand. It should be something representing a business value on a unit of time. For example, in an integration application, how many entities were moved from one system to another per using of time.
+  * [Throughput](https://www.cdnetworks.com/en/news/6-critical-web-application-performance-metrics-to-consider/4257). This is very tight to the application at hand. It should be something representing a business value on a unit of time. For example, in an integration application, how many entities were moved from one system to another per unit of time.
```

Comment:
per unit of time. => per time period?

Reply:
It is more or less the same, but it could be clearer to say time period.